js/modules/k6/k6.go

```diff
@@ -59,7 +59,17 @@ func (*K6) Group(ctx context.Context, name string, fn goja.Callable) (goja.Value
 	state.Group = g
 	defer func() { state.Group = old }()
 
-	return fn(goja.Undefined())
+	tags := make(map[string]string)
+	tags["group"] = g.Path
```

Comment:
Just add the tags inline when you send out the sample. No need for a dedicated variable when there's nothing changing them.